app/scss/modal.scss

```diff
@@ -0,0 +1,21 @@
+.overlay {
+  background: rgba(0,0,0,0.6);
```

Comment:
opacity? white will default surely? so opacity: 0.6; will do the same in less characters.